lib/src/snapping_sheet_widget.dart

```diff
@@ -238,7 +238,7 @@ class _SnappingSheetState extends State<SnappingSheet>
       ),
     );
     _animationController.reset();
-    _animationController.forward();
+    await _animationController.forward();
```

Comment:
I think that we should return the `TicketFuture` from the forward call so that the user can access it from the controller. This would enable the user of the controller to have more control, such as handling cancel events from the `TicketFuture`

Reply:
Sure. I thought of the `TickerFuture` as more of an implementation detail that was subject to change in the future, so I cast it down. Your point about cancellation makes sense, though.